CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 Bugfixes:
 - Fix CollectionResolver to allow propagation of inline dependencies (@dvdwouwe, #562)
 - Allow DefaultNamingSubSystem derivatives to invalidate the cache which was accidently removed in 5.1.0 (@nativenolde, #569)
+- Fix dictionary bug where reference to list components didn't work (@ni-mi, #575)
```

Comment:
Could you change this to refer to the fact this is about XML configuration, right now it is unclear especially that most people are doing everything in code.

Reply:
Changed the description to reflect the fact that it is about XML configuration.